scripts/set_vars.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+set -eo pipefail
+
+APP_ROOT_PATH=$(cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && cd .. && pwd)
+
+export APP_ROOT_PATH="${APP_ROOT_PATH}"
+export SC_WEB_PATH="${APP_ROOT_PATH}"
+export SCRIPTS_PATH="${APP_ROOT_PATH}"/scripts
```

Comment:
If those are scripts of sc-web specifically rather than the app itself, it should be something like this:
```suggestion
export SCRIPTS_PATH="${SC_WEB_PATH}"/scripts
```